src/Shortcodes/FileShortcodeProvider.php

```diff
@@ -237,9 +238,9 @@ protected static function find_error_record($errorCode)
      */
     public static function getCacheKey($params, $content = null)
     {
-        $key = SSViewer::config()->get('global_key');
-        $viewer = new SSViewer_FromString($key);
-        $globalKey = md5($viewer->process(ArrayData::create([])) ?? '');
+        $key = SSTemplateEngine::config()->get('global_key');
+        $engine = SSTemplateEngine::create();
+        $globalKey = md5($engine->renderString($key, ViewLayerData::create([])));
```

Comment:
Explicitly use `SSTemplateEngine` instead of getting whatever's registered against `TemplateEngine`  because this config is specific to `SSTemplateEngine` and is explicitly ss template syntax.